### ISSUE TITLE
Standard for most of recorded music is 44khz instead of 40khz

### ISSUE THREAD

**Issue by kalomaze** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#233]
I've been told contentvec hubert was trained at 44khz, but there are 3 options in RVC for 40khz, 48khz, and 32khz. This confuses me, as the standard for most recorded music is 44khz, which means you must always pick 48khz to avoid this 10% quality loss, and yet it is not the default option. Ideally, there would be a 44khz pretrained weight replacing the 32khz one? So it would be a default of 44, with 40, and 48 as alternative options. It would be interesting too if a method would be implemented to scan the wavs for the 'true' khz based on the metadata and adjust training weights as necessary, during pre-processing.

**Comment by kalomaze** [comment]
It looks like the change in the above pull request was rolled back after I showed off an example of a 40khz beating a 44khz model. Defenitely needs more testing, because maybe the default config for 48khz is suboptimal? There is also a chance 48khz audio works best with 48khz, but 44khz works better with 40khz because you're not exporting beyond the 'real frequency', which guarantees consistency.

**Comment by yxlllc** [comment]
I guess the reason is that the pre-trained model of rvc is trained with the VCTK dataset with a sampling rate of 48khz.

**Comment by RVC-Boss** [comment]
"contentvec hubert was trained at 44khz"
but the data should be downsampled to 16khz first. 
So for the performance: 44khz is the same as 40khz or 48khz
